spec/swagger.yaml

```diff
@@ -6141,18 +6150,22 @@ paths:
                   key: "job.not.found"
                   message: "Job {translationJobUid} not found for project {projectId}."
         500: *Error500ResponseDefinition
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST \
+            'https://api.smartling.com/jobs-batch-api/v1/projects/$smartlingProjectId/batches' \
+            -H "Authorization: Bearer $smartlingToken"  \
+            -d '{
```

Comment:
there could be escaping issues here, lets try it out and see if its work. @hchaps i can help with this.

Reply:
fixed.